modernomad/settings/local.py

```diff
@@ -1,16 +1,21 @@
+import sys
 from .common import *  # noqa
-from .common import INSTALLED_APPS
+from .common import INSTALLED_APPS, MIDDLEWARE_CLASSES
 
 DEBUG = True
-INSTALLED_APPS = INSTALLED_APPS + [
-    'debug_toolbar'
-]
+TESTING_MODE = 'test' in sys.argv
+DEV_MODE = DEBUG and not TESTING_MODE
+
+if DEV_MODE:
```

Comment:
These environment variables took a bit to get my head round. Perhaps we can avoid the extra `DEV_MODE` variable by just having `if not TESTING_MODE` here? That perhaps better explains the intention when reading the code -- don't load the debug toolbar when running tests.